Let's take time to learn about some of the most common exceptions, what each one means, and how to handle them as you start your writing your own code. In order for you to try and test the exception causing scenarios, use the `try{}` and `catch{}` blocks in each scenario.

Have you experienced the "ArrayIndexOutOfBoundsException" before? It is thrown when you try to access an array through an index that doesn't exist in the array. For example, calling an index that is less than 0 or greater than or equal to the length of the array will cause this error. Take a look at the code below.

